src/main/java/com/fasterxml/jackson/core/json/JsonGeneratorImpl.java

```diff
@@ -245,6 +245,10 @@ public JacksonFeatureSet<StreamWriteCapability> getWriteCapabilities() {
         return JSON_WRITE_CAPABILITIES;
     }
 
+    public IOContext _getIoContext() {
```

Comment:
Nit:

* Capitalization `getIOContext()`
* If it needs to be public, no leading underscore; only internal methods to use that

Reply:
Changed. Ideally would avoid altogether but... oh well.